src/app/startups/applications/create/page.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
+
+import AddApplications from "@/components/addApplications";
+
+export default function Create() {
+  return (
+    <div className="container mx-auto p-4">
```

Comment:
If a lot of views will have this class it shoul be a component called Layout or something like that